README.md

```diff
@@ -38,16 +39,16 @@ kubectl apply -f ./kubemacpool.yaml
 
 ### Opting-in to kubemacpool service
 
-On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
+On releases v0.8.4 and above, kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
```

Comment:
We should not refer to z-stream and above. It is not clear if 0.9.0 is newer or older than that. I think we can omit the mention of version completely.

Reply:
ack